core/benches/bench.rs

```diff
@@ -48,13 +49,49 @@ fn clear(test_values: &[u8]) {
 
 fn pop_all(test_values: &[u8]) {
     let mut vec = vec_from_slice(&test_values);
-
     while vec.len() > 0 {
         vec.pop();
     }
```

Comment:
this could be improved to
```
while let Some(_) = vec.pop() {}
```
this way it won't directly access the `vec.len` field.